src/lib/y2storage/storage_manager.rb

```diff
@@ -144,6 +145,15 @@ def update_staging_revision
       @staging_revision += 1
     end
 
+    # Probes all storage devices.
+    #
+    # Invalidates the probed and staging devicegraph.
+    def probe
+      storage.probe
```

Comment:
Until we kill all the legacy code (to support direct access to the `Storage` classes), we will also need this here:
```ruby
@y2probed = nil
@y2staging = nil
```